lib/web_ui/lib/painting.dart

```diff
@@ -351,6 +351,8 @@ abstract class Image {
 
   List<StackTrace>? debugGetOpenHandleStackTraces() => null;
 
+  ColorSpace get colorSpace => ColorSpace.sRGB;
```

Comment:
Looks like all concrete subclasses are implementing this getter. I think this means that the getter in the base class can be abstract.

Reply:
The reason all of the other places had to implement it themselves is because they are using `implements` instead of `extends`.  I don't know if there was anyone that was using `extends`, but if they were, I wouldn't want them to have to implement this property.